selenium_tests/publish_courses/publish_courses_base_test_case.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
+from urlparse import urljoin
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_tests.canvas_account_admin_tools_base_test_case \
+    import AccountAdminBaseTestCase
+from selenium_tests.publish_courses.page_objects.main_page import MainPageObject
+
+
+class PublishCoursesBaseTestCase(AccountAdminBaseTestCase):
+
+    def setUp(self):
+        super(PublishCoursesBaseTestCase, self).setUp()
+
+        #  Test course URL
+        self.base_url = settings.SELENIUM_CONFIG['canvas_base_url']
+        self.test_course_relative_url = settings.SELENIUM_CONFIG[
+            'publish_courses']['test_course']['relative_url']
+        self.test_course_url = urljoin(self.base_url,
+                                    self.test_course_relative_url)
+
+        # Test data for use in tests
+        self.term = settings.SELENIUM_CONFIG['publish_courses']
```

Comment:
Why is this called `self.term`? I find that a bit confusing. Maybe `self.config` or `self.test_settings`?